internal/oci/container.go

```diff
@@ -125,11 +125,13 @@ type ContainerState struct {
 // at any future time.
 // imageName, if set, is _some_ name of the image imageID; it may have NO RELATIONSHIP to the users’ requested image name.
 // imageID is nil for infra containers.
```

Comment:
Please document _precisely_ what is the value / purpose of this parameter.
> someRepoDigest, if set, is _some_ repo@_some_ digest of the image imageID; it may have NO RELATIONSHIP to the users’ requested image name (and, which should be fixed eventually, may be a repo@digest combination which has never existed on a registry.

or

> containerStatusImageRef, if set, is a value intended to be returned in ContainerStatus [and other CRI APIs??]; it’s the caller’s responsibility to confirm with CRI semantics

or something.

We know these things now, so let’s some poor soul 5 years later the trouble of finding out.

Reply:
Alright, added the comment and changed the variable name